media-libs/harfbuzz/harfbuzz-9.0.0.recipe

```diff
@@ -117,7 +138,7 @@ BUILD()
 		--datadir=$dataDir \
 		--bindir="$commandBinDir" \
 		-Dcairo=disabled \
-		-Dglib=disabled \
+		-Dglib=enabled \
```

Comment:
this was disabled in 24e31ffce283420266b7a9af3d47dd9e19e966cd for some reason

Reply:
IIRC it was disabled for a reason (eg not have a dependeny on glib)?

Reply:
yeah the commit message often isn't really helpful...

Reply:
I enabled it here because I was trying to port GTK4 and it needs a Harfbuzz with this option set; personally given that glib is a very standard package to have I can't see why having a dependency on it here would be a problem

Reply:
Harfbuzz is used in app_server, and so, all dependencies have to be shipped with Haiku, and it creates some complications to update them. So we try to keep dependencies to a minimum in these packages.

Also, glib provides things largely redundant with our existing standard libraries, so including it in Haiku base install would be a bit against the spirit of the whole project, I guess (to try to provide an unified system with a single API to do each thing). But different people may have different visions of what Haiku is and wether this is acceptable.

Reply:
https://docs.gtk.org/gtk4/building.html doesn't list harfbuzz as a dependency. https://www.linuxfromscratch.org/blfs/view/12.2/x/gtk4.html doesn't either.

Reply:
Looking at the old package I have here it seems to list it in packagelist (lost the recipe down the road it seems)
```
	lib:libgraphene_1.0 >= 0.1000.8
	lib:libharfbuzz >= 0.60821.0
	lib:libharfbuzz_gobject >= 0.60821.0
```

Reply:
> https://docs.gtk.org/gtk4/building.html doesn't list harfbuzz as a dependency. https://www.linuxfromscratch.org/blfs/view/12.2/x/gtk4.html doesn't either.

Weird; when I was compiling it it wanted to include `hb_glib.h`, which only gets installed when you set glib=true

Reply:
All this for https://github.com/GNOME/gtk/commit/1f7783224b7bc6bd8ca4b0f906e3ee542cc249db